File: EASY/app/001 - Name Age User/challenge_001.py

name = raw_input("Hello. What is your name?")
age = int(raw_input("How old are you?"))
reddit = raw_input("What is your Reddit username?")

text = "Your name is %s, you are %d years old, and your username is %s." % (name, age, reddit)

file = open("output_test.txt", "w")
file.write(text)
file.close

print text

File: EASY/app/001 - Name Age User/challenge_001.php
<?php

echo "Hello. What is your name?";
$name = chop(fgets(STDIN));
echo "How old are you?";
$age = chop(fgets(STDIN));
echo "What is your Reddit username?";
$reddit = chop(fgets(STDIN));

$text = "Your name is $name, you are $age years old, and your username is $reddit.";

$file = fopen("output_test.txt", "w");
fwrite($file, $text);
fclose($file);

echo $text;